docs/swarm/swarm-mode.md

```diff
@@ -0,0 +1,176 @@
+<!--[metadata]>
++++
+title = "Run Docker Engine in swarm mode"
+description = "Run Docker Engine in Swarm mode"
+keywords = ["guide, cluster management, swarm mode"]
+advisory = "rc"
+[menu.main]
+identifier="initialize-swarm-guide"
+parent="engine_swarm"
+weight=12
++++
+<![end-metadata]-->
+
+# Run Docker Engine in swarm mode
+
+When you first install and start working with Docker Engine (Engine), swarm mode
+is disabled by default. When you're working outside of swarm mode, you create
+images and run containers. If you need a refresher on containers and images, see
+the [Engine user guide](../userguide/intro.md).
+
+There are two ways to run the Engine in swarm mode:
+
+* create a new swarm, covered in this article.
+
+* join an existing cluster, covered in Join an existing swarm (forthcoming).
+
+When you run the Engine in swarm mode on your local machine, you can create and
+test a services based upon images you've created or other available images. In
+your production environment, swarm mode provides a fault-tolerant platform with
+cluster management features to keep your services running and available.
+
+If you haven't already, read through the [swarm mode key concepts](key-concepts.md)
+and try the [swarm mode tutorial](swarm-tutorial/index.md).
+
+## Create a cluster
+
+When you run the command to create a swarm, the Docker Engine starts running in
+a swarm mode. Run [`docker swarm init`](/engine/reference/commandline/swarm_init.md)]
+to create a single-node swarm on the current node.
+
+The `docker swarm init` command does the following:
+
+* switches the Docker Engine on the current node into swarm mode.
+
+* creates a swarm named `default`.
+
+* designates the current node as a manager node for the swarm.
+
+* configures the manager to listen on an active network interface on port 2377.
+
+    >**Note**: Docker recommends you use the `--listen-addr` flag to
+    specify precisely the IP address and port where the manager should
+    listen.
+
+* Creates a root CA for the swarm and implements the default security policy.
+   More in depth documentation about about mutual TLS Authentication is forthcoming.
+
+* Implements a raft consensus model for managing the swarm state and the desired
+state for services.
+
+More topics are forthcoming to help you plan for deploying services into
```

Comment:
In general, I would remove sentences like this that state an intention to expand this documentation in the future.  They aren't helpful to the reader.  Best to just put forward what we have and enhance it as we are able.